docs/variability.md

```diff
@@ -83,3 +83,4 @@ The median Lighthouse score of 5 runs is twice as stable as 1 run, and tools lik
 
 - [Lighthouse Variability and Accuracy Analysis](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit?usp=sharing)
 - [Throttling documentation](./throttling.md)
+- [Why is my Lighthouse score different from PageSpeed Insights?](https://www.debugbear.com/blog/why-is-my-lighthouse-score-different-from-pagespeed-insights)
```

Comment:
🥇

Reply:
I am a tad concerned about making this a "blessed" explanation if it's in our docs, but I also want lots of 👀 on it too sooooooo... 👍